### DISCUSSION TITLE
Restore only data or config from backup

### DISCUSSION THREAD

**Discussion by gbertolottiKS** [nextcloud/all-in-one#2879] [Ideas] [answered]
Hi.
Is there a way to restore from a backup only the data or the configuration?

From our experience, when a server crash occurred, it's mostly due to a bad configuration/update/installation, not a file problem. So it would be very powerful if it could be possible to restore from the backup a previous configuration, but leaving all the data untouched (to have the latest version of the working data).

Thanks in advance.

**Comment by szaimen** [comment, marked as answer]
Hi, this is not really possible with Nextcloud as the data, database, Nextcloud code and co must be restored at the same time as you otherwise risk that the filecache in Nextcloud gets out-of-sync with the data and other things as well which would be catrastophal if you have any kind of encryption enabled in Nextcloud and can cause unexpected sideffects. So this is not going to be implemented by us.

As a sidenote: whenever restoring the backup, we adjust the data-fingerprint as recommended in Nextclouds restore docs which should usually allow the desktop client to upload all changes that were made in the meantime.

**Reply by gbertolottiKS** [reply]
Thanks.
At least now we know: as always, thanks for your fast answer.

But I take your sidenote suggestion to make another question: we are a team of 20 people circa, during the years more than 150 GB of files have been created, mostly documents.
These files should be visible and editable by anyone of us: to achieve these we made a shared giant group folder with write permission which contains all the documents.
The fact is that often we have the need to edit them locally, but, as far as we know, to do that it is necessary that everyone sync that folder with a local one, using the desktop app. 
Is this feasible? We have to sync 20 times that folder with each client (using virtual files). Do you think it could be a problem for the Nextcloud server to handle all these different connections ?
Any suggestion?

Thanks in advance

**Reply by szaimen** [reply]
> Is this feasible? We have to sync 20 times that folder with each client (using virtual files). Do you think it could be a problem for the Nextcloud server to handle all these different connections ?

This should work and should not be a problem.
